src/development/vortex/development/core/engine/validator/base_validator.py

```diff
@@ -382,44 +395,26 @@ def __call__(self, *args, **kwargs):
                     targets=targets,
                     last_index=last_index
                 )
+                # Perform validation loss calculation if supported
+                # Some model behave differently on training on eval, validation loss calculation might failed
+                if self.criterion:
+                    try:
+                        raw_results = self.raw_model_output
+                        batch = list(zip(image,targets))
+                        _,targets = self.collate_fn(batch)
+                        targets = targets.to(raw_results.device)
+                        batch_loss = self.criterion(raw_results,targets)
+                        val_loss += batch_loss.detach()
+                        self.raw_model_output = None
+                    except Exception as e:
+                        print(e)
```

Comment:
probably better to not print anything here, because if it fails once, it will always fail for all the dataset. have a warning in the end of validation would be better

Reply:
Noted